core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -336,6 +339,54 @@ public static void main(String[] args) throws Exception {
         }
     }
 
+    static void reportCrashFiles(CairoConfiguration configuration, Log log) {
+        final CharSequence dbRoot = configuration.getRoot();
+        final FilesFacade ff = configuration.getFilesFacade();
+        final int maxFiles = configuration.getMaxCrashFiles();
+
+        NativeLPSZ name = new NativeLPSZ();
+        try (
+                Path path = new Path().of(dbRoot).slash$();
+                Path other = new Path().of(dbRoot).slash$()
+        ) {
+            int plen = path.length();
+            AtomicInteger counter = new AtomicInteger(0);
+            FilesFacadeImpl.INSTANCE.iterateDir(
+                    path,
+                    (pUtf8NameZ, type) -> {
+                        if (Files.notDots(pUtf8NameZ)) {
+                            name.of(pUtf8NameZ);
+                            if (Chars.startsWith(name, "hs_err_pid")) {
```

Comment:
What happens if you have table named `hs_err_pid`?
I think we should verify that `type` is not directory.

Reply:
there is bigger issue, if table is created:

```sql
create table "hs_err_pid1.log"(a int);
```

the crash log will not be written

Reply:
fixed in https://github.com/questdb/questdb/pull/2346/commits/96d82d06c17c01f0f605e6708135c60d262085e1